src/components/UserAuth/Login.css

```diff
@@ -0,0 +1,57 @@
+/*-------------------------Login page(card)---------------*/
+
+.login-check:invalid {
+	border: var(--Mak-colors-red-500) solid 1px;
+	border-radius: 1.5rem;
+}
+/* .login-check:valid {
+	border: var(--Mak-colors-green-500) solid 1px;
+	border-radius: 1.5rem;
+} */
```

Comment:
Can remove the commented code if not required.

Reply:
Okay I'll do that thank's Vrinda for PR review